src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner.vue

```diff
@@ -157,11 +189,69 @@ export default {
 			this.open = true;
 		}
 		this.calculateAmountRaised();
+
+		// GROW-230
+		// if on homepage and EXP-GROW-230-Sept2020 is on,
+		// set a dynamic class to use for css overrides
+		if (this.$route.name === 'homepage' && this.forceDismissOverlayExperiment.version === 'variant-b') {
+			this.$refs.pageOverlay.classList.add('overlay-shown');
```

Comment:
This is fine for now but no ref is needed as you could just toggle a boolean data variable that applies the class directly to your div using the Vue paradigm.